### ISSUE TITLE
Add label to query return

### ISSUE THREAD

**Issue by 007nh** [RareMint/raremint#3]
## Context
Please add the string "Current sale price" as a label for the query `assetForSale` return value

## Definition of Done:
Query should return `Current sale price: 150`

## Estimation Breakdown
- Dev work: .2 day
- Testing: .2 day
- Review: 0

Story Points reference:
2

**Comment by bmacer** [comment]
to clarify, is the issue here that the api.query.collectibles.assetForSale query returns an integer (for example, 100), where you want it to return {"current_sale_price": 100} ?

**Comment by 007nh** [comment]
@bmacer correct. I think it should return a KV pair with a label. Can also add a check to validate whether or not an asset is for sale. 

`{
"current_sale_price": 100,
"isListed": boolean
}`

Let me know your thoughts on that approach.

**Comment by bmacer** [comment]
That's good, can do.  Only issue I'd say is that the current mechanics are to add/remove from this data set when listing/unlisting, so there would never be a case (currently) where isListed would be false.  Can imagine scenarios where that might change so probably worth including now anyway

**Comment by 007nh** [comment]
Agreed, somewhat future-proofing this query
